README.md

```diff
@@ -25,23 +25,33 @@ There are multiple task files available in this repository with different object
 
 ### setup.yaml
 
-This task file includes a single task `k3d-test-cluster` which sets up a k3d cluster running the `uds-core-istio-dev` package. This provides a baseline cluster with Istio and the UDS Core Pepr capabilities necessary to test against.
+This task file includes two tasks:
+
+- `k3d-test-cluster` sets up a k3d cluster running the `uds-core-istio-dev` package. This provides a baseline cluster with Istio and the UDS Core Pepr capabilities necessary to test against.
+- `registry-login` performs a registry login and retry a max of 10 times at a desired retry interval (in seconds). It reads in environment variables from the system executing the task to use for the registry login parameters.  The environment variable inputs are:
```

Comment:
```suggestion
- `registry-login` performs a registry login and retry a max of 10 times at a desired retry interval (in seconds). It reads in variables for the task to use for the registry login parameters.  The variable inputs are:
```
Think we can drop environment? In testing I think `--set` works fine and would prefer to document assuming that is used?